typo fix and allow bundle install

alx3dev opened the pull request:
There was an typo with 'observr'. I also added MIT license, since there wasn't any, together with Gemfile and few other files needed to publish ruby gem (there is console for development and setup for dependencies). Now one can download git and run 'bundle install', or you can push gem to allow 'gem install jenncad'. If you don't want to change anything, just fix that typo.
Happy New Year!

Commit:
typo fix and allow bundle install

alx3dev committed:
Update console

jglauche commented:
Obervr wasn't actually typo, but it's a really old dependency:
https://rubygems.org/gems/observr/versions/1.0.5

Have you tested if "observer" is a replacement for it? 

Next, my gemspec shows LGPL3. Mind if we keep that?

alx3dev commented:
Sure, keep what you want. I didn't checked for observr, but I am going to do that now.

Commit:
restore observr gem and add LGPLv3 license

alx3dev commented:
I didn't checked for 'observr', I just added 'observer' as dependency :) I removed it and added text license LGPLv3. So basically I just added gemfile and license, and edited specifications.

jglauche commented on Gemfile:
```diff
@@ -0,0 +1,8 @@
+
+source "https://rubygems.org"
+
+# Specify your gem's dependencies in jenncad-fork.gemspec
```
please rename to 
jenncad.gemspec

jglauche commented on CHANGELOG.md:
```diff
@@ -0,0 +1,7 @@
+
+## [0.1.0] - 2022-01-06
+
+- Fixing typo "require 'observer'"
```
please remove

alx3dev replied:
So many mistakes, I worked on two devices. Deleted `observer` and `jenncad-fork`, in `Gemfile` and `CHANGELOG.md`

jglauche commented on CHANGELOG.md:
```diff
@@ -0,0 +1,7 @@
+
+- Fixing typo "require 'observer'"
+- Allow "bundle install" from source
+- prepared to be released as ruby gem (added MIT license)
```
please change to LGPLv3

Commit:
Changelog and Gemfile fix

alx3dev commented:
I want to use this gem to create models, and to show/animate using OpenGL and ruby. I need more time to check it's code and difference between crystal-scad. It's all just for testing purposes.
